lib/cocina_display/dates/date.rb

```diff
@@ -340,12 +344,13 @@ class << self
         # @param date [Date] The date to format.
         # @param precision [Symbol] The precision to format the date at, e.g. :month
         # @param allowed_precisions [Array<Symbol>] List of allowed precisions for the output.
-        #   Options are [:day, :month, :year, :decade, :century].
+        #   Options are [:day, :month, :year, :decade, :century, :unknown].
         # @note allowed_precisions should be ordered by granularity, with most specific first.
         def format_date(date, precision, allowed_precisions)
           precision = allowed_precisions.first unless allowed_precisions.include?(precision)
-
           case precision
+          when :unknown
+            "Unknown"
```

Comment:
Is there a standard somewhere for how to format this? I'm fine with just writing "Unknown" but it kind of feels like a question for @arcadiafalcone or a metadata person